vizro-ai/src/vizro_ai/dashboard/_response_models/components.py

```diff
@@ -63,18 +63,20 @@ def create(self, model: BaseChatModel, all_df_metadata: AllDfMetadata) -> Compon
                 from vizro_ai import VizroAI
 
                 vizro_ai = VizroAI(model=model)
-                result = vizro_ai._run_plot_tasks(
+                result = vizro_ai.plot(
```

Comment:
We need a better fallback here if the plot failed with Validation error. Now it doesn't get caught and breaks the dashboard building process.
![Screenshot 2024-09-04 at 11 31 51](https://github.com/user-attachments/assets/a5fcbc29-bb9c-4b18-844a-fc15b408ba17)

Reply:
I think replace the `DebugFailure` with `ValidationError` should do the trick
```
        except ValidationError as e:
            logger.warning(
                f"""
[FALLBACK] Build failed for `Component`, returning default values.
Reason: {e}
Relevant prompt: {self.component_description}
"""